src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
```

Comment:
I mean that scan_proper should NOT be used at all for formats. Only one hookspec should be needed for formats that handles the the format code for the option and the callable that will return this format.